src/components/footer/style.module.css

```diff
@@ -1,9 +1,79 @@
 .root {
-  padding: 1rem;
-  min-height: 160px;
-  background: var(--token-color-palette-neutral-700);
+  align-items: center;
+  background: var(--token-color-palette-neutral-0);
   color: var(--token-color-palette-neutral-600);
   display: flex;
-  align-items: center;
-  justify-content: center;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  padding: 16px 0;
+  position: relative;
+
+  /* before element acts as full-width border,
+     no matter the container size*/
+  &::before {
+    background: var(--token-color-neutral-border-primary);
+    content: '\200B';
+    height: 1px;
+    left: 0;
+    left: 50%;
+    overflow: hidden;
+    position: absolute;
+    top: 0;
+    transform: translateX(-50%);
+    width: 100%;
+    width: 200vh;
+  }
+}
+
+.logo {
+  position: relative;
+  margin: 16px 40px 16px 0;
+
+  & svg {
+    /* default inline display results
+     in misalignment with footer links */
+    display: block;
+  }
+}
+
+.links {
+  position: relative;
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  /* !important needed to prevent list styles from leaking in
+     from docs layout content CSS */
+  padding: 0 !important;
+  margin: 0 !important;
```

Comment:
Am I imagining it, or has this scenario popped up in a few recent PRs? It feels like we should try to scope the content CSS if possible so we don't need to do this.

Reply:
Not imaging it at all - it has come up since #48 merged. One option here would be to try to write some clever CSS to scope the content CSS - eg one thing I've done in the past in `react-content` is like `ul:not([class])`. But that has pitfalls, and feels like a stopgap...

I think a longer-term solution here would be using MDX custom components with accompanying CSS module styles, rather than targeting HTML primitives [as we do now](https://github.com/hashicorp/dev-portal/blob/assembly-ui-v1/src/layouts/docs/docs-layout.module.css).

Not sure when we can fit this in - but definitely agree we should address ASAP. Have created [a task for this](https://app.asana.com/0/1201010428539925/1201653123138729/f) since we don't have one yet 👍 Happy to switch gears on that and hold on this PR if we want to avoid introducing these new `!important`s.

Reply:
I think for now we could move the HTML selectors into a nested class that only wraps the rendered MDX so they aren't bleeding into footer 💭 
![Screen Shot 2022-01-12 at 11 50 00 AM](https://user-images.githubusercontent.com/1289701/149194884-9854db07-e643-4d29-a574-1e56a294b47c.png)

Reply:
I'd be happy to make that Asana task my next piece of code work! My load is a little lighter until I start the downloads view, and I can fit this task in before I start that especially since that page will be affected by this issue. I'm spending the rest of my day working on writing Asana task for the downloads view and then I should be able to start the task tomorrow. How does that sound?

Reply:
On board with both of these - for now, added a quick container in order to avoid introducing new `!important`s in `footer` 👍